### PR TITLE
Banner image and header 

### PR DESCRIPTION
**Pull request by chair28980**
Addresses https://github.com/pi0neerpat/chili-kitchen/issues/9

https://www.figma.com/file/xFPT4QEZ02kol0BnY66FwW/CHILI-Kitchen?node-id=84%3A458

### CODE REVIEW

File: src/components/common/Layout.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ import '../../styles/app.css'
 const DefaultLayout = ({ children, bodyClass, isHome }) => {
 
     const site = {
-        "title": 'Bon Apetit!',
+        "title": 'Chili Kitchen',
```
Comment (pi0neerpat) [inline src/components/common/Layout.js]: Please capitalize CHILI

File: src/styles/app.css [hunk 2]
```diff
@@ -663,13 +663,13 @@ h6 {
...
 .site-head {
-    padding-top: 20px;
+    /* padding-top: 20px; */
     padding-bottom: 20px;
     color: #fff;
     background: var(--color-base);
     background-position: center;
-    background-size: cover;
-    background-image: url("../images/cook.jpg");
+    background-size: contain;
```
Comment (pi0neerpat) [inline src/styles/app.css]: Issue on mobile with repeated image, likely due to this change
image

Also on desktop for pages besides index 
image

File: src/components/common/Layout.js [hunk 3]
```diff
@@ -51,17 +51,20 @@ const DefaultLayout = ({ children, bodyClass, isHome }) => {
...
+                                <div><div className="button-container"></div>
                                 <div className="site-banner">
-                                    <h1 className="site-banner-title">{site.title}</h1>
-                                    <p className="site-banner-desc">{site.subtitle}</p>
-                                </div> :
+                                    {/* empty div for space */}
+                                </div>
+                                <div className="site-banner">
+                                    <a href="/about" className="site-banner-title">About the project</a>
+                                </div>
```
Comment (pi0neerpat) [inline src/components/common/Layout.js]: We should continue using the existing code here `site.title`, `site.subtitle`

Are we using react-helmet?

**Review by pi0neerpat — CHANGES_REQUESTED** [review]

**Comment by pi0neerpat** [comment]
Also, please adjust the font to match the designs

File: src/components/common/Layout.js (hunk 3 above)
Comment (chair28980) [reply src/components/common/Layout.js]: @pi0neerpat Indeed react-helmet is in play

I pushed an updated before reading your review here, I'll readjust this to the original.

**Comment by chair28980** [comment]
To do:

1. Fonts
2. site.title and site.subtitle code
3. banner image on about page
4. mobile resp issues

**Comment by pi0neerpat** [comment]
Just a few more changes. 

- [x] Lets move the "About' button to the menu instead, so it doesn't block the text here.
- [x] Remove the connect button 

image

**Review by pi0neerpat — APPROVED** [review]

**Pull request merged**